### ISSUE TITLE
How to check if a cacheKey exists?

### ISSUE THREAD

**Issue by Jordan1122** [Baseflow/flutter_cached_network_image#700]
## How to check if a cacheKey exists?

Is it possible to check if an image is in the cache if I only have the cacheKey? It should work on every platform (web, mobile).

Thank you for your help!

**Comment by Colton127** [comment]
I have the same question/problem. I download SD by default, and HD when the full image is opened. I then want the HD loaded for there on for the preview, because it is already cached. 

Can't find a way to do this through the plugin. The key can be stored manually and then checked later on.

**Comment by Jordan1122** [comment]
Exactly, this is my main use-case too. I pass only the cacheKey to a new page, and I want to use Hero animation. 
If there is no solution I will use what you just mentioned, but It seemed strange not to have this feature.
